src/shared/components/instances.tsx

```diff
@@ -79,8 +79,9 @@ export class Instances extends Component<any, any> {
                     <h4 class="col">{domain}</h4>
                     <h4 class="col text-right">
                       <i>
-                        {numToSI(users_active_month)} {i18n.t("users")} /{" "}
-                        {i18n.t("month")}
+                        {i18n.t("users_active_per_month", {
+                          count: users_active_month,
+                        })}
```

Comment:
To format plurals right, you supply both `count: num`, and `formattedCount: numToSi(num)`

https://github.com/LemmyNet/lemmy-translations/blob/main/translations/en.json#L7

Reply:
Okay fixed.